app/views/layouts/application.html.erb

```diff
@@ -6,6 +6,9 @@
   <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
   <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
   <%= csrf_meta_tags %>
+  <link rel="icon" type="image/png" href="favicon-32x32.png" sizes="32x32" />
+  <link rel="icon" type="image/png" href="favicon-16x16.png" sizes="16x16" />
```

Comment:
We'll want to do a different favicon on this branch, since it's the anonymous one. The Rails way is: 
<%= favicon_link_tag 'favicon.ico' %>
Place the favicon.ico image in /app/assets/images/. 
Example: https://github.com/forplayplay/GCLibraryCollection/commit/83e7827134be79f91e3394a4d16517a56216cebf